### DISCUSSION TITLE
how to make  a non xml stage

### DISCUSSION THREAD

**Discussion by ghost** [YoshiCrafter29/YoshiCrafterEngine#99] [Q&A]
it crashes when i put in a image

**Comment by ghost** [comment]
![image](https://user-images.githubusercontent.com/97402522/158863684-a7695acd-27e9-4f1a-9b57-ec42f383d95a.png)

**Comment by ghost** [comment]
and i tried making a empty xml 
![image](https://user-images.githubusercontent.com/97402522/158864213-220e1df1-2d90-4f68-ab96-f70dcdc28236.png)

**Comment by ErenKaya31** [comment]
you can just copy the other stage background and set the offsets why did you not think of that

**Reply by ghost** [reply]
?

**Reply by ghost** [reply]
wydm

**Reply by ghost** [reply]
i legit just copied the assets from dave and bambi to my mod by grabbing it
